packages/components/src/Actions/ActionDropdown/ActionDropdown.snapshot.test.js

```diff
@@ -33,6 +35,21 @@ describe('ActionDropdown', () => {
 		expect(wrapper).toMatchSnapshot();
 	});
 
+	it('should render a button with label and a filled dropdown when immutable items', () => {
+		// given
+		const props = {
+			id: 'dropdown-id',
+			label: 'related items',
+			items: immutableItems,
+		};
+
+		// when
+		const wrapper = renderer.create(<ActionDropdown {...props} />).toJSON();
+
+		// then
+		expect(wrapper).toMatchSnapshot();
```

Comment:
Could use a toMatchObject between the two json (one with immutable the other with POO)